nca/NetworkConfig/NetworkConfigQueryRunner.py

```diff
@@ -101,7 +101,8 @@ def run_query(self, cmd_line_flag=False):
         comparing_err = 0
         self.output_configuration.print_query_output(query_output, formats)
         if self.expected_output_file is not None:
-            if self.query_name in {'ConnectivityMapQuery', 'SemanticDiffQuery'}:
+            if self.query_name in {'ConnectivityMapQuery', 'SemanticDiffQuery'} or\
+                    self.output_configuration.FullExplanation:
```

Comment:
I hesitated to generalize this, since when `FullExplanation` is False we can not predict what is the one example that will be printed (so I wanted to avoid using expectedOutput when full explanation is not used).

Also, to avoid using it with queries that were not touched in this PR (queries derived from `NetworkConfigQuery`) - maybe their outputs needs to be sorted too

Reply:
OK, got it. Leave it as-is then.